loadtests/loadtest/rooms.py

```diff
@@ -39,14 +42,26 @@ def setupRoom(self):
 
     def create_room(self):
         self.hawk_room_owner = self.hawk_auth
+        data = {
+            "roomName": "UX Discussion",
+            "expiresIn": 1,
+            "roomOwner": "Alexis",
+            "maxSize": MAX_NUMBER_OF_PEOPLE_JOINING
+        }
+
+        if random.randint(0, 100) < PERCENTAGE_OF_ROOM_CONTEXT:
+            del data['roomName']
+            data['context'] = {
+                "value": b64encode(os.urandom(1024)).decode('utf-8'),
```

Comment:
That's cool. However, it seems we don't check that the values are present afterwards when getting the information back? If we want to do so, then using a random value might not be what we want.